experiments/operational_domain/operational_domain_siqad.cpp

```diff
@@ -0,0 +1,187 @@
+//
+// Created by marcel on 08.08.23.
+//
+
+#include "fiction/algorithms/simulation/sidb/operational_domain.hpp"  // operational domain computation algorithms
+#include "fiction/io/read_sqd_layout.hpp"                             // reader for SiDB layouts
+#include "fiction/io/write_operational_domain.hpp"                    // writer for operational domains
+#include "fiction/types.hpp"                                          // pre-defined types suitable for the FCN domain
+#include "fiction/utils/truth_table_utils.hpp"                        // truth tables helper functions
+#include "fiction_experiments.hpp"                                    // experiment class
+
+#include <fmt/format.h>                   // string formatting
+#include <kitty/dynamic_truth_table.hpp>  // truth tables
+
+#include <array>
+#include <cstdint>
```

Comment:
We don't need this, right?

Reply:
Yes, we do. We use `std::size_t` and `uint64_t`.